### DISCUSSION TITLE
sharethis sticky bouton

### DISCUSSION THREAD

**Discussion by triclic** [AmauriC/tarteaucitron.js#1113] [General]
Salut à tous,
comment faire pour intégrer sharethis sticky dans tarteaucitron ?
car le code et du style :
<script type='text/javascript' src='https://platform-api.sharethis.com/js/sharethis.js#property=xxxxxxxxxxxxx&product=sticky-share-buttons' async='async'></script>

Merci

**Comment by AmauriC** [comment]
Je m'en occupe rapidement :)
https://github.com/AmauriC/tarteaucitron.js/issues/1123

**Reply by triclic** [reply]
Merci
